imagehash.py

```diff
@@ -320,7 +321,7 @@ def whash(image, hash_size = 8, image_scale = None, mode = 'haar', remove_max_ha
 
 
 
-def colorhash(image, binbits=3):
+def colorhash(image, binbits=3, ignore_icc=False):
```

Comment:
use a positive variable name here to avoid the double negative. For example: `apply_icc_profile=True`